homeassistant/components/sentry/__init__.py

```diff
@@ -44,15 +51,98 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
 
     # https://docs.sentry.io/platforms/python/logging/
     sentry_logging = LoggingIntegration(
-        level=logging.INFO,  # Capture info and above as breadcrumbs
+        level=logging.WARNING,  # Capture warning and above as breadcrumbs
```

Comment:
in case anyone ever looks back, this is def something that'd be nice to add config for in a follow-up patch. after using this in prod for several months i could imagine wanting to dial it back (or increase it) at various times.

Reply:
See TODO section in the PR description.

Reply:
Resolved in #38975